### PR TITLE
(feat) Add option to define position of ticks

### PR DESCRIPTION
**Pull request by MelleB**
- Add `ticks_positions` option (array)
- Current limitation: Does not work with logarithmic scale.

### CODE REVIEW

**Comment by MelleB** [comment]
http://jsfiddle.net/v8htqnfz/1/

File: js/bootstrap-slider.js [hunk 1]
```diff
@@ -204,8 +204,26 @@
 			linear: {
 				toValue: function(percentage) {
 					var rawValue = percentage/100 * (this.options.max - this.options.min);
-					var value = this.options.min + Math.round(rawValue / this.options.step) * this.options.step;
+					if (this.options.ticks_positions) {
```
Comment (rovolution) [inline js/bootstrap-slider.js]: This large block of code looks very similar to lines 240-256...Anyway we can refactor that logic into a single method?

Comment (MelleB) [inline js/bootstrap-slider.js]: @rovolution Thanks for the review. I tried that as well, but the small differences make it hard to capture it in a single method without severely hurting readability. I'd prefer to leave this as it is.

Comment (rovolution) [inline js/bootstrap-slider.js]: @MelleB sounds good. could you rebase against master and push back up? Thanks!

**Comment by MelleB** [comment]
Rebase + fix label positioning when using `ticks_positions`

**Comment by rovolution** [comment]
@MelleB When you try to move the slider in Example 14, it ends up breaking the container.

image

**Comment by MelleB** [comment]
@rovolution Should be fixed now!

**Comment by rovolution** [comment]
@MelleB could you rebase again? Promise i will merge this time before asking for another rebase :)

**Comment by MelleB** [comment]
@rovolution No problem :-) Just a heads up: It'll take a couple of weeks
before I'll be able to though...

Sent from mobile
Op 22 mrt. 2015 00:06 schreef "Rohit Kalkur" <notifications@github.com>:

**Comment by rovolution** [comment]
no worries

**Comment by MelleB** [comment]
Should be good to go!

**Comment by seiyria** [comment]
That is a lot of commits, would you mind rebasing it?

**Comment by rovolution** [comment]
rebase would be nice

**Comment by MelleB** [comment]
Rebased

**Pull request merged**